### PR TITLE
Feature/asf 397 update foundation

### PR DESCRIPTION
**Pull request by pvhee**

### CODE REVIEW

**Comment by pvhee** [comment]
@nunoveloso care to review?

File: modules/teaser_standard/teaser_standard.strongarm.inc [hunk 1]
```diff
@@ -69,10 +69,19 @@ function teaser_standard_strongarm() {
       'revision' => array(
         'custom_settings' => FALSE,
       ),
-      'teaser_tiny' => array(
-        'custom_settings' => TRUE,
-      ),
       'teaser_basic' => array(
+        'custom_settings' => FALSE,
+      ),
+      'list_teaser' => array(
```
Comment (jolidog) [inline modules/teaser_standard/teaser_standard.strongarm.inc]: @pvhee anything list_ should not be in this repo.

File: modules/teaser_standard/teaser_standard.features.field_instance.inc [hunk 2]
```diff
@@ -88,10 +88,19 @@ function teaser_standard_field_default_field_instances() {
         'weight' => 5,
       ),
       'teaser_tiny' => array(
-        'label' => 'hidden',
-        'settings' => array(),
-        'type' => 'hidden',
-        'weight' => 4,
+        'label' => 'above',
```
Comment (jolidog) [inline modules/teaser_standard/teaser_standard.features.field_instance.inc]: @pvhee the teaser_tiny should not contain the short description field.

File: modules/teaser_standard/teaser_standard.features.field_instance.inc [hunk 3]
```diff
@@ -168,8 +176,13 @@ function teaser_standard_field_default_field_instances() {
...
         'label' => 'hidden',
-        'settings' => array(),
-        'type' => 'hidden',
+        'module' => 'title',
+        'settings' => array(
+          'title_class' => '',
+          'title_link' => '',
+          'title_style' => '',
+        ),
+        'type' => 'title_linked',
```
Comment (jolidog) [inline modules/teaser_standard/teaser_standard.features.field_instance.inc]: @pvhee teaser_tiny should not have the field_teaser_icon.

File: modules/teaser_standard/teaser_standard.features.field_instance.inc [hunk 4]
```diff
@@ -317,8 +342,9 @@ function teaser_standard_field_default_field_instances() {
       ),
       'teaser_tiny' => array(
         'label' => 'above',
+        'module' => 'link',
         'settings' => array(),
-        'type' => 'hidden',
+        'type' => 'link_default',
```
Comment (jolidog) [inline modules/teaser_standard/teaser_standard.features.field_instance.inc]: @pvhee This should also remain hidden.

File: modules/teaser_standard/teaser_standard.features.field_instance.inc [hunk 5]
```diff
@@ -205,14 +218,18 @@ function teaser_standard_field_default_field_instances() {
       'default' => array(
         'label' => 'hidden',
         'module' => 'atom_reference',
-        'settings' => array(),
-        'type' => 'thumbnail_xlarge',
+        'settings' => array(
+          'link' => 0,
+        ),
+        'type' => 'title',
```
Comment (jolidog) [inline modules/teaser_standard/teaser_standard.features.field_instance.inc]: @pvhee this one should remain thumbnail_xlarge for the type.

**Pull request merged**